db/seeds.rb

```diff
@@ -0,0 +1,61 @@
+require 'csv'
+
+DRIVER_FILE = Rails.root.join('db', 'seed_data', 'drivers.csv')
+puts "Loading raw driver data from #{DRIVER_FILE}"
+
+CSV.foreach(DRIVER_FILE, :headers => true) do |row|
+  driver = Driver.new
+  driver.id = row['id']
+  driver.name = row['name']
+  driver.vin = row['vin']
+  puts "Created driver: #{driver.inspect}"
+  driver.save!
```

Comment:
I really like when we have a seed file that keeps track of how many records have been added successfully and how many failed. I think this is a good practice to show the students. Any opposition to updating this to be a bit more forgiving if errors occur upon save?

Reply:
Just looked at this comment; the seed file actually already keeps track of how many get saved.

I've actually never written a seed file before! Would the best way to write this just be:
```
driver_failures = []
...
successful = driver.save
if !successful
  driver_failures << driver
...

puts "Added #{Driver.count} driver records"
```

?

Reply:
Yes something like that is good! This is one example from what we did with books seeds for C7 --
 just another way to indicate success 

https://github.com/Ada-C7/stacks-book-app/blob/master/db/seeds.rb